tests/unit/projection-test.js

```diff
@@ -619,22 +624,24 @@ module('unit/projection', function(hooks) {
     });
 
     test('Setting a projection updates the base-record and other projections', function(assert) {
-      let excerpt = this.records.projectedExcerpt;
+      let preview = this.records.projectedPreview;
```

Comment:
why this change?

Reply:
The excerpt doesn't whitelist the `chapter-1` property, which breaks. I figured `preview` suits the rest of the test conditions.